plugins/kvscheduler/api/txn_options.go

```diff
@@ -34,6 +34,11 @@ const (
 	// the context.
 	retryCtxKey
 
+	// default parameters for the *retry* txn option
+	defaultRetryPeriod = time.Second
```

Comment:
These could perhaps be exported variables instead of constants to allow changing the defaults globally.